nixos/modules/services/web-apps/readeck.nix

```diff
@@ -0,0 +1,91 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+with lib;
```

Comment:
We usually do not add this prefix any more, would it be possible to only import what's needed ?